MIGRATION.md

```diff
@@ -0,0 +1,65 @@
+# Migration Guide
+
+## v11.0.0
+
+Version 11.0.0 of orb-tools is composed of a major re-write that greatly changes and improves the way orbs are deployed, makes use of CircleCI's [dynamic configuration](https://circleci.com/docs/2.0/dynamic-config/), and can even automatically test for best practices.
+
+### Notable changes:
+
+- **Removed the 90 day limit**
+  - Previously, the configuration relied on calling a `dev:alpha` tag of the orb for testing. If it had been over 90 days since the orb was last published, this would result in an error in the CircleCI pipeline. Users would have to manually publish their orb locally to re-start the ci pipeline
```

Comment:
Replaced it as such:

> Previously, the configuration relied on calling a `dev:alpha` tagged version of the orb for testing. Due to "dev" tags on CircleCI being ephemeral with a 90-day life span, if it had been over 90 days since the orb was last published this would result in an error in the CircleCI pipeline. Users would have to manually publish their orb locally to re-start the ci pipeline